### DISCUSSION TITLE
Having trouble with generators in config

### DISCUSSION THREAD

**Discussion by jhclouse** [clickbar/laravel-magellan#97] [Q&A]
Thank you for the library. It's great!

One problem I'm having, though, is getting anything but the binary representation out of the models by default. I can obviously do a raw query and use ST_AsText or ST_AsGeoJson. But I tried setting the `string_generator` in `config/magellan.php` to use the GeoJson generator by default and as far as I can tell it doesn't do anything. Am I misunderstanding what the config is supposed to do?

**Comment by saibotk** [comment]
What version are you using?

Jeah the config is used to convert the object to string when it is casted for example. But there might be a bug, can you try to reproduce this on the latest version? And maybe try some simple conversion in tinker and send the code snippet? Otherwise ill take a look at this in some weeks as im currently busy with finishing university

**Reply by jhclouse** [reply]
Thanks for the reply. I was using the latest version as of a few weeks ago but I haven't worked on this part of the code lately. I'll check on it when I get a chance and report back. Good luck with your studies!
